package.json

```diff
@@ -49,5 +50,10 @@
     "fancy-log": "^1.3.3",
     "prettier": "^2.4.1"
   },
-  "prettier": {}
+  "prettier": {
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": false,
+    "singleQuote": true
+  }
```

Comment:
This doesn't seem to have fixed the issue. Prettier still won't complain if you reformat. Given the formatting change has been reverted, I would revert this config and leave it as i for this PR.

Reply:
taken out 👍